codegenerator/cli/templates/dynamic/codegen/src/TestHelpers.res.hbs

```diff
@@ -232,7 +235,7 @@ module {{contract.name.capitalized}} = {
   {{#each contract.codegen_events as | event |}}
   module {{event.name}} = {
     @genType
-    let processEvent = EventFunctions.makeEventProcessor(
+    let processEvent: EventFunctions.eventProcessor<Types.{{contract.name.capitalized}}.{{event.name}}.event> = EventFunctions.makeEventProcessor(
```

Comment:
Had to add the `eventProcessor` type to be able to specify block and transaction types in a good way.